cookies/templates/dashboard.html

```diff
@@ -60,55 +60,55 @@
       <div class="panel panel-default">
         <div class="panel-heading">
           <h4 class="panel-title">
-            <a class="toggle" data-toggle="collapse" href="#collapse-data">Giles Upload Statistics</a>
+            <a class="toggle" data-toggle="collapse" href="#collapse-data">Giles Upload Statistics ( {{ user }} )</a>
           </h4>
         </div>
         <div id="collapse-data" class="panel-collapse collapse in">
               <div class="row text-center data-row">
                   <div class="col-sm-3">
-                      <a class="btn btn-hg" href="{% url "giles-log" %}?state={{ PENDING }}&o=-updated">
+                      <a class="btn btn-hg" href="{% url "giles-log" %}?state={{ PENDING }}&o=-updated&created_by={{request.user.id}}">
```

Comment:
is there a reason you append 'created_by' here? it doesn't seem to be used.

Reply:
For the dashboard for individual stats, I added this so that can users can see their own data. If created_by is not added then it will show data for all of the users.